subsys/bluetooth/controller/ll_sw/ull_slave.c

```diff
@@ -99,6 +100,19 @@ void ull_slave_setup(memq_link_t *link, struct node_rx_hdr *rx,
 	win_offset = sys_le16_to_cpu(pdu_adv->connect_ind.win_offset);
 	conn_interval_us = interval * 1250U;
 
+	if (0) {
+#if defined(CONFIG_BT_CTLR_ADV_EXT)
+	} else if (adv->lll.aux) {
+		if (adv->lll.phy_s & BIT(2)) {
+			win_delay = 3;
+		} else {
+			win_delay = 2;
+		}
+#endif
+	} else {
+		win_delay = 1;
+	}
+
```

Comment:
Is this 1, 2 and 3 BT Spec constants? define them in pdu.h or lll_tim_internal.h